src/topology/local_homeomorph.lean

```diff
@@ -507,7 +507,7 @@ begin
     exact this.congr (λy hy, by simp [e.left_inv hy.2]) (by simp [e.left_inv hx]) }
 end
 
-/-- Continuity at a point can be read under right composition with a local homeomorphism if a
+/-- Continuity at a point can be read under leftt composition with a local homeomorphism if a
```

Comment:
Typo copy-pasted.

Reply:
Even not copy-pasted. I guess I am very deterministic when I make typos!